File: app/src/main/java/com/sortedqueue/programmercreek/activity/HelpActivity.java
package com.sortedqueue.programmercreek.activity;

import android.app.Activity;
import android.content.Context;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.sortedqueue.programmercreek.CreekApplication;
import com.sortedqueue.programmercreek.R;

import uk.co.chrisjenx.calligraphy.CalligraphyContextWrapper;


public class HelpActivity extends AppCompatActivity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_help);
	}
	@Override
	protected void attachBaseContext(Context newBase) {
		super.attachBaseContext(CalligraphyContextWrapper.wrap(newBase));
	}

	@Override
	protected void onResume() {
		super.onResume();
		CreekApplication.getInstance().setAppRunning(true);
	}

	@Override
	protected void onPause() {
		super.onPause();
		CreekApplication.getInstance().setAppRunning(false);
	}
}

File: app/src/main/java/com/sortedqueue/programmercreek/activity/HelpActivity.kt
package com.sortedqueue.programmercreek.activity

import android.app.Activity
import android.content.Context
import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import com.sortedqueue.programmercreek.CreekApplication
import com.sortedqueue.programmercreek.R

import uk.co.chrisjenx.calligraphy.CalligraphyContextWrapper


class HelpActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_help)
    }

    override fun attachBaseContext(newBase: Context) {
        super.attachBaseContext(CalligraphyContextWrapper.wrap(newBase))
    }

    override fun onResume() {
        super.onResume()
        CreekApplication.instance.isAppRunning = (true)
    }

    override fun onPause() {
        super.onPause()
        CreekApplication.instance.isAppRunning = (false)
    }
}
